.github/workflows/main-workflow.yml

```diff
@@ -0,0 +1,27 @@
+name: Test and Release Main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    uses: ./.github/workflows/test-base.yml
+  test_cypress:
+    needs: test
+    uses: ./.github/workflows/test-cypress.yml
+    secrets:
+       chromaticProjectToken: ${{ secrets.CHROMATIC_CYPRESS_PROJECT_TOKEN }}
+  test_playwright:
+    needs: test
+    uses: ./.github/workflows/test-playwright.yml
+    secrets:
+       chromaticProjectToken: ${{ secrets.CHROMATIC_PLAYWRIGHT_PROJECT_TOKEN }}
+  release:
+    needs: test
+    uses: ./.github/workflows/release.yml
+    secrets:
+      githubToken: ${{ secrets.GITHUB_TOKEN }}
+      npmToken: ${{ secrets.NPM_PUBLISH_TOKEN }}
```

Comment:
Woohoo, I assume this means we'll automatically release the changeset stuff instead of needing a 2nd PR each time, is that correct?

Reply:
No, sorry! The changeset release workflow always opens a second PR so that it can be approved and merged in explicitly.

Reply:
Not for this PR, but that'd be great to have automated, since it'll happen frequently.